src/components/charts/TempChart.vue

```diff
@@ -273,11 +278,14 @@ export default class TempChart extends Mixins(BaseMixin, ThemeMixin) {
     tooltipFormatter(datasets: any) {
         let output = ''
 
+        //window.console.log('tempchartFix', datasets)
+
         const mainDatasets = datasets.filter((dataset: any) => dataset.seriesName.endsWith('-temperature'))
         if (datasets.length) {
             let outputTime = datasets[0]['axisValueLabel']
+            //window.console.log('tempchartFix', outputTime)
             outputTime = outputTime.substring(outputTime.indexOf(' '))
-            let theme = $vuetify.theme.dark ? 'theme-dark' : ''
+            const theme = this.$vuetify.theme.dark ? 'theme-dark' : ''
```

Comment:
this was fixed in pr #1706 ?

Reply:
yes, because i started with the 1706 branch with this fix. i will update this branch...